templates/launch.html

```diff
@@ -6,9 +6,10 @@ <h1>Launch a survey</h1>
 
 <form action="" method="POST" xmlns="http://www.w3.org/1999/html" onsubmit="validateForm()">
 
+    <strong><u>Launch by Schema Name</u></strong>
```

Comment:
Unarticulated annotation is used for styling text differently, misspelled words etc. Can be confused with a link, but up to you.

Reply:
Should be fine for a dev frontend that will soon get a facelift anyway.